pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -239,9 +239,75 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Monitor the cloud-credential-operator-config configmap to check if operator is re-enabled.
+	// Check if attempts were made to delete the credentials requests when the operator was disabled.
+	// We have to have to reconcile all those credential requests.
+	configmapMapFn := handler.ToRequestsFunc(
```

Comment:
Kudos Akhil, this is better than what we talked about in scrum yesterday. Why restart the pod when we can just requeue when we detect a change we care about.